docs/introduction/faq-and-common-issues.md

```diff
@@ -67,7 +67,8 @@ See how to install Shopsys Framework in production and how to proceed when deplo
 
 ## How to set up the administration with a different locale/language (e.g. Czech)?
 The administration uses `en` locale by default.
-If you want to switch it to the another locale, override the method `getAdminLocale()` of the class `Shopsys\FrameworkBundle\Model\Localization\Localization`.
+If you want to switch it to the another locale, override a parameter `shopsys.admin_locale` in your `parameters.yml` configuration.
```

Comment:
Just a grammar tweak: I think you can either "override the parameter" (as the parameter must be there to be overridden) or "set a parameter" (as it's a new parameter in that file):
```suggestion
If you want to switch it to the another locale, set a parameter `shopsys.admin_locale` in your `parameters.yml` configuration.
```